iocore/net/quic/QUICAckFrameCreator.cc

```diff
@@ -104,6 +104,15 @@ QUICAckFrameManager::_on_frame_acked(QUICFrameInformationUPtr &info)
   this->_ack_creator[index]->forget(ack_info->largest_acknowledged);
 }
 
+void
+QUICAckFrameManager::_on_frame_lost(QUICFrameInformationUPtr &info)
+{
+  ink_assert(info->type == QUICFrameType::ACK);
+  int index = QUICTypeUtil::pn_space_index(info->level);
+  // when ack frame lost. Force to refresh the frame.
+  this->_ack_creator[index]->refresh_frame();
```

Comment:
We're not refreshing *frame* anymore. Maybe `refresh_state`?

Reply:
will do .